docs/man-pages/htcondor.rst

```diff
@@ -3,84 +3,169 @@
-    Used in conjunction with the *--resource* flag.
- **--email**
-    Email address to receive notification messages.
-    Used in conjunction with the *--resource* flag.
- **--skip-history**
-    Passed to the *status* subcommand to skip checking history
-    if job not found in the active job queue.
 
-Job Set Options
----------------
+Job Verbs
+---------
+
+ **htcondor job submit submit_file**
+     Takes as an argument a submit file in the *condor_submit* job submit
+     description language, and places a new job in an Access Point
+
+     **htcondor job submit Options**
+     
+          **htcondor job submit --resource resource_type submit_file**
+            Resource type used to run this job. Currently supports ``Slurm`` and ``EC2``.
+            Assumes the necessary setup is complete and security tokens available.
+          **htcondor job submit --runtime runtime_in_seconds submit_file**
+            Amount of time in seconds to allocate resources.
+            Used in conjunction with the *--resource* flag.
+          **htcondor job submit --email address submit_file**
+            Email address to receive notification messages.
+            Used in conjunction with the *--resource* flag.
+    
+    
+ **htcondor job status**
+     Takes as an argument a job id in the form of clusterid.procid,
+     and returns a human readable presentation of the status
+     of that job.
+
+     **job status option**
+
+      **htcondor job status --skip-history job.id**
+
+        Passed to the *status* verb to skip checking history
+        if job not found in the active job queue.
+
+ **htcondor job resources**
+     Takes as an argument a job id in the form of clusterid.procid,
+     and returns a human readable presentation the machine resource
+     used by this job.
+
+Jobset Options
+--------------
 
  **--allusers**
-    Passed to the *list* subcommand to show job sets from all users
+    Passed to the *list* verb to show job sets from all users
     rather than just the current user.
  **--nobatch**
-    Passed to the *status* subcommand to display the status of
+    Passed to the *status* verb to display the status of
     individual job clusters within a job set
  **--owner=USERNAME**
-    Passed to the *status* or *remove* subcommand to act on job sets
+    Passed to the *status* or *remove* verb to act on job sets
     submitted by the specified user instead of the current
     user. Using this option to *remove* job sets requires superuser
     permissions.
  **--skip-history**
-    Passed to the *status* subcommand to skip checking history
+    Passed to the *status* verb to skip checking history
     if job clusters are not found in the active job queue.
 
+Eventlog Verbs
+--------------
+
+ **htcondor eventlog read logfile**
+     Takes as an argument an event log to process.  It may be the per-job or
+     per-jobset eventlog, which was specified by the *log = some_file* in the
```

Comment:
If "eventlog" is the preferred nomenclature, consider all my previous "event log"s corrected.